packages/legacy/core/App/localization/fr/index.ts

```diff
@@ -187,6 +187,16 @@ const translation = {
         "OpenSettings": "Ouvrir les paramètres",
     },
     "PINCreate": {
+        "Explainer": {
+          "PrimaryHeading": "Secure your Wallet",
+          "PINReminder": "On the following screen you will be asked to set up a strong <b>personal identification number</b> (PIN).",
+          "WhyNeedPin": {
+            "Header": "Why you need a PIN",
+            "Paragraph": "Your PIN is an important security feature:",
+            "ParagraphList1": "You can use your PIN instead of biometrics to open your wallet",
+            "ParagraphList2": "Or use your biometrics to open your wallet, and if your biometrics change change on your device you will need to use the PIN to authenticate your wallet"
```

Comment:
Because this is the French translation file and these are in English, please trail each with a `(FR)` so that we know they need to be translated. You'll also need to add these to the portuguese `(PT-BR)` translation file. Sorry I didn't catch this in my first review